DocumentStores/CosmosDb/Halforbit.DataStores.DocumentStores.CosmosDb/Implementation/CosmosDbDataStore.cs

```diff
@@ -47,11 +51,7 @@ public class CosmosDbDataStore<TKey, TValue> :
 
         readonly Lazy<Container> _container;
         
-        readonly string _connectionString;
-        
-        readonly string _databaseId;
-        
-        readonly string _containerId;
```

Comment:
These should not get removed. While they arent used by the implementation, these fields are used by the builder integration tests to verify that the proper values are propagated. See `Halforbit.DataStores.IntegrationTests.FluentBuilderTests.CosmosDb()`.

Reply:
Fixed in 6c69825f2eebd4a96da9e4b233094040d38fdfb9